### PR TITLE
Remove login/logout code along with the admin and user sections.

### PR DESCRIPTION
**Pull request by robsiemb**
They are all basically dead code at this point.

### CODE REVIEW

File: content/leftmenu.inc [hunk 1]
```diff
@@ -13,7 +13,6 @@
 </nav>
 
 <?php include("googlesearchinput.inc"); ?>
-<?php include_once("loginform.inc"); ?>
```
Comment (madisonscott) [inline content/leftmenu.inc:16]: The nav scrollbar is shrinking 🙌

File: content/user/auth_general.php [hunk 2]
```diff
@@ -1,10 +0,0 @@
-<?php
-	//used to authenticate users for gallery3
-  include("../../util.inc");
-   
-  if (isset($_COOKIE["cmubuggy_auth"])){
```
Comment (madisonscott) [inline content/user/auth_general.php:5]: Is this the "single sign on" bit? i.e. Will users still be able to log in to the gallery and forum separately with their gallery and forum specific creds?

File: .htaccess [hunk 3]
```diff
@@ -22,7 +22,6 @@ Redirect 302 /fantasy https://docs.google.com/forms/d/e/1FAIpQLSeiUE_9vJIAAo67G_
 RedirectMatch 302 (?i)/RD20 https://cmu.zoom.us/j/91610341394
 
 #top level sections
-RewriteRule		^admin[/]*$						index.php?s=admin				[NC,L]
```
Comment (madisonscott) [inline .htaccess:25]: Could we redirect `/admin` to the Wordpress dashboard at `/news/wp-admin`?

File: content/user/auth_general.php (hunk 2 above)
Comment (robsiemb) [reply content/user/auth_general.php:5]: As far as I can tell, yes -- I don't have any main site creds and was able to log into the forum just fine.

As for the gallery, it wasn't clear to me if the logins worked at all so I didn't try? :)

File: .htaccess (hunk 3 above)
Comment (robsiemb) [reply .htaccess:25]: Sure

**Pull request merged**